### ISSUE TITLE
Small UI bug where the spinner in Vision tool is not centered

### ISSUE THREAD

**Issue by phuctm97** [sanity-io/sanity#6775]
**Describe the bug**

When vision tool is loading, the spinner is ugly displayed in the top left corner.

**To Reproduce**

1. Go to Vision Tool
2. Wait for it to load and see the spinner

**Expected behavior**

The spinner should be centered.

**Which versions of Sanity are you using?**

```
@sanity/cli (global)    3.43.0 (up to date)
@sanity/asset-utils      1.3.0 (up to date)
@sanity/icons           2.11.8 (up to date)
@sanity/image-url        1.0.2 (up to date)
@sanity/studio-secrets   3.0.0 (up to date)
@sanity/ui              2.1.13 (up to date)
@sanity/vision          3.43.0 (up to date)
sanity                  3.43.0 (up to date)
```

**What operating system are you using?**

macOS

**Which versions of Node.js / npm are you running?**

Node.js 20, pnpm 9

**Comment by ninaandal** [comment]
Thank you for reporting - I've added this to our backlog 👍

**Comment by ninaandal** [comment]
Fixed in [v3.47.0](https://github.com/sanity-io/sanity/releases/tag/v3.47.0) ✨
